react/components/organisms/chat/chat.js

```diff
@@ -59,7 +61,7 @@ export class Chat extends PureComponent {
     };
 
     scrollToEnd = () => {
-        this.scrollViewComponent.scrollToEnd({ animated: true });
+        this.scrollViewComponent.scrollToEnd({ animated: this.props.scrollAnimated });

```

Comment:
Who is calling `scrollToEnd`?

Reply:
React `ScrollView` component: https://github.com/ripe-tech/ripe-components-react-native/blob/a10530729b7bb7463f36af2877e5cf8f3e96203a/react/components/organisms/chat/chat.js#L127
and inside the `_onNewMessage`method: https://github.com/ripe-tech/ripe-components-react-native/blob/a10530729b7bb7463f36af2877e5cf8f3e96203a/react/components/organisms/chat/chat.js#L79

Reply:
Does this need to be a prop then? Isn't it enough to be a `scrollToEnd` argument that you manipulate in `chat.js`?

Reply:
I added this prop so I can disable the scroll animation when the chat messages are being loaded. It's related to the enhancement you asked me to do (https://github.com/ripe-tech/ripe-robin-revamp/pull/267#issuecomment-866939952)

Reply:
Ok rename to `animateScrollToEnd` then, and it's good to go